### ISSUE TITLE
When switching from 'muuri' to 'zoomin' storyview, the columns and cards on open kanban boards overlap

### ISSUE THREAD

**Issue by AshRubigo** [BurningTreeC/tiddlywiki-muuri-kanban#2]
Note: Dragging the columns and cards a bit stops the overlapping. Seems related to JavaScript rather than CSS.

**Environment:**
Version: Main github branch.
Browser: Firefox 90.0.2.
TiddlyWiki: 5.1.23, 5.2.0-prerelease.

**Steps to reproduce:**

1. Switch to 'muuri' storyview.
1. Create a kanban board tiddler with some columns. Leave this tiddler open.
2. Create some tiddlers to be used as card tiddlers.
3. Manually add the card tiddlers to the list field of a column tiddler.
1. Switch to 'zoomin' storyview.
1. See how columns overlap each other, and cards overlap each other.

**Comment by BurningTreeC** [comment]
Yes @AshRubigo , that's a known issue. It has to do with the zoomin storyview animating while the muuri storyview has already done its positioning of the items. I'll see if I can fix this but I will need time for that

**Comment by BurningTreeC** [comment]
What I've found now is that this is FireFox specific. It doesn't happen in chromium-based browsers or webkit browsers
